doc-deploy-dev/action.yml

```diff
@@ -315,6 +317,7 @@ runs:
           sed -i 's/action="search.html"/action="version\/dev\/search.html"/g' index.html
           sed -i 's|script\.src = ".*_static/js/meilisearch_theme_wrap\.js"|script.src = "version/dev/_static/js/meilisearch_theme_wrap.js"|g' index.html
           sed -i 's|const SEARCH_FILE = ".*_static/search.json";|const SEARCH_FILE = "version/dev/_static/search.json";|g' index.html
+          sed -i 's|var searchPath = ".*_static/search.json";|var searchPath = "version/dev/_static/search.json";|g' index.html
```

Comment:
I hope you will remember to delete this one too :P

Reply:
<img src="https://media2.giphy.com/media/7xZAu81T70Uuc/giphy.gif"/>

Reply:
Looooool

Reply:
> I hope you will remember to delete this one too :P

yes, i will open an issue for this for us to remember